Submissions/dummygithubusername/Assignment-1/assignment-1.py

```diff
@@ -0,0 +1,55 @@
+from hashlib import sha256
+import time
+
+class block():
+	def __init__(self, timestamp, data, previousHash = ''):
+		self.timestamp = timestamp
+		self.data = data
+		self.previousHash = previousHash
+		self.hash = self.calculateHash()
+
+	def calculateHash(self):
+		#todo
+		self.hash = hashlib.sha256(self.previousHash + self.data)
+		#return hash--------------------
+		return self.hash
+
+class blockchain():
+
+	def __init__(self):
+		self.chain = [self.createGenesis()]
+
+	def createGenesis(self):
+		return block(time.ctime(), "Genesis Block", "00000")
+
+	def mineBlock(self, data):
+		# current Time = time.ctime()
+		current time = time.ctime()
```

Comment:
i guess it is supposed to a wrong variable

Reply:
self.timestamp = time.ctime()
I guess this is it.